upstream-community-operators/gpu-operator/1.0.0/gpu-operator.v1.0.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,207 @@
+      mediatype: image/png
+  install:
+    spec:
+      clusterPermissions:
+        - rules:
+            - apiGroups: security.openshift.io
+              resources:
+                - securitycontextconstraints
+                - securitycontextconstraints/finalizers
+              verbs: '*'
+            - apiGroups: ""
+              resources: nodes
+              verbs: '*'
+            - apiGroups: sro.openshift.io
+              resources: '*'
+              verbs: '*'
+            - apiGroups: ""
+              resources: secrets
+              verbs:
+                - get
+                - watch
+                - list
+          serviceAccountName: gpu-operator
+      deployments:
+        - name: gpu-operator
+          spec:
+            replicas: 1
+            selector:
+              matchLabels:
+                name: gpu-operator
+            strategy: {}
+            template:
+              metadata:
+                labels:
+                  name: gpu-operator
+              spec:
+                containers:
+                  - command:
+                      - gpu-operator
+                env:
+                  - name: WATCH_NAMESPACE
+                    valueFrom:
+                      fieldRef: {fieldPath: 'metadata.annotations[''olm.targetNamespaces'']'}
+                  - name: POD_NAME
+                    valueFrom:
+                      fieldRef: {fieldPath: metadata.name}
+                  - name: OPERATOR_NAME
+                    value: gpu-operator
+                image: REPLACE_IMAGE
```

Comment:
You need to set an image, this is not replaced or set by OLM.

Reply:
Oops, good catch. There was a Makefile that copies a template that I was using, a lot like how special-operator sets its up. but now that I'm manually doing it I missed the `sed` commands the Makefile was doing.